streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -602,7 +603,8 @@ public static StreamThread create(final InternalTopologyBuilder builder,
         log.info("Creating restore consumer client");
         final Map<String, Object> restoreConsumerConfigs = config.getRestoreConsumerConfigs(threadClientId);
         final Consumer<byte[], byte[]> restoreConsumer = clientSupplier.getRestoreConsumer(restoreConsumerConfigs);
-        final StoreChangelogReader changelogReader = new StoreChangelogReader(restoreConsumer, userStateRestoreListener, logContext);
+        final Duration pollTime = Duration.ofMillis(config.getLong(StreamsConfig.POLL_MS_CONFIG));
```

Comment:
As above: should we check for `"restore.consumer.poll.ms"` ?

Reply:
I guess not, following Guozhang's comment.